generate_ics.py

```diff
@@ -10,12 +10,31 @@
 
 import build_event
 
+import argparse
+import getpass
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-i", "--input")
+parser.add_argument("-o", "--output")
+args = parser.parse_args()
+
 DEBUG = False
 GENERATE_ICS = True
 TIMETABLE_DICT_RE ='([0-9]{1,2}):([0-9]{1,2}):([AP])M-([0-9]{1,2}):([0-9]{1,2}):([AP])M'
 timetable_dict_parser = re.compile(TIMETABLE_DICT_RE)
 
-OUTPUT_FILENAME = "timetable.ics"
+if not os.path.exists("data.txt"):
```

Comment:
Hey, that's a whole lot of lines do something very simple. Just keep it very very simple. I am sorry to keep requesting changes, but there's too much cruft here. :slightly_smiling_face: Here's a way to do it:

```py
INPUT_FILE = args.input if args.input else "data.txt"
if not os.path.exists(INPUT_FILE):
    print "Input file %s does not exist." % (INPUT_FILE)
    os.exit(1)
```